README.md

```diff
@@ -63,6 +63,15 @@ The advantages of this approach are:
 
 - Assurance that all the code repositories are tested in this same way.
 
+CI scripts also provide a convenient way for other Kata repositories to
+install software. The preferred way to use these scripts is to invoke `make`
+with the corresponding `install-` target. For example, to install crio you
+would use:
+
+`$ make -C <path-to-this-repo> install-crio`
```

Comment:
Can you use the "long-hand" triple-backtick syntax for these two examples as they render differently so good to be consistent with the other docs and code snippets?:

```
    ```
    $ make -C <path-to-this-repo> install-crio
    ```
```

Reply:
done :)